src/shell/navigation/Navigation.tsx

```diff
@@ -11,44 +15,39 @@ interface INavigationState {
     title: string
 }
 
-export class Navigation extends React.Component<INavigationProps, INavigationState> {
-
-    public state = {
-        navigationExpanded: false,
-        title: 'Navigation'
-    }
+const navigation = props => {
+    const [ navigationExpanded, setNavigationExpanded ] = useState(false);
+    const [ title, setTitle ] = useState('Navigation');
 
-    public toggleNavigationHandler = () => {
-        this.setState(state => {
-            return { navigationExpanded: !state.navigationExpanded };
-        })
-    }
+    public const toggleNavigationHandler = () => {
+        setNavigationExpanded(!navigationExpanded);
+    };
 
-    public render(): React.ReactNode {
         return (
-            <div className={`navigation ${this.state.navigationExpanded ? `navigation_expanded` : ``}`}>
-                <button className='navigation__toggle' type='button' onClick={this.toggleNavigationHandler}>
-                    <span className='navigation__toggle-logo'>FT</span>
-                    <span className={`navigation__toggle-text ${this.state.navigationExpanded ? `navigation__toggle-text_visible` : ``}`}>Finance Track</span>
+            <div className={`styles.navigation ${navigationExpanded ? `styles.navigation_expanded` : ``}`}>
+                <button className='styles.navigation__toggle' type='button' onClick={toggleNavigationHandler}>
+                    <span className='styles.navigation__toggle-logo'>FT</span>
+                    <span className={`styles.navigation__toggle-text ${navigationExpanded ? `styles.navigation__toggle-text_visible` : ``}`}>Finance Track</span>
                 </button>
                 <CSSTransition
-                    in={this.state.navigationExpanded}
+                    in={navigationExpanded}
                     timeout={300}
                     classNames='navigation__routes'
                 >
                     {state => (
                         <div className='navigation__routes'>
-                            {this.props.routes.map((route, index) => {
+                            {props.routes.map((route, index) => {
                                 return (
                                     <React.Fragment key={index}>
-                                        <NavigationItem {...route} navigationExpanded={this.state.navigationExpanded} expandedState={state} />
+                                        <NavigationItem {...route} navigationExpanded={navigationExpanded} expandedState={state} />
                                     </React.Fragment>
                                 )
                             })}
                         </div>
                     )}
                 </CSSTransition>
             </div>
-        )
-    }
-}
\ No newline at end of file
+        );
+};
+
+export default navigation;
```

Comment:
let's use named imports in the begining of the component, like 
export const Navigation: FunctionComponent<...>.....